### ISSUE TITLE
Feature suggestion: Add Taskview button

### ISSUE THREAD

**Issue by ShadowEO** [ADeltaX/MobileShell#3]
Decided to throw your shell on my x86 tablet, looks wonderful but I noticed there's no good way to switch applications if I didn't have the keyboard connected. Perhaps add a Taskview button to the right of the search button? Or maybe a swipe gesture across the navigation bar to switch to the next or previous app?

On that note, perhaps also add a space to expose system tray icons, I can see the ability to manage the legacy notification icons (systray) being a huge thing for running Windows 10 on any device that can support Win32 based applications, such as OneDrive, EventGhost, Google Drive, etc.

As I said, the shell as it is now, is pretty impressive. Looking forward to reading your code! :D

**Comment by ADeltaX** [comment]
Taskview is already here.
Longpress the back button (or right click) to open the taskview.

For the swipe gesture... I'll think about it (not that hard to achieve but I need to see how laggy it will be, lol)

Systray it will be implemented in a future.

**Comment by ShadowEO** [comment]
Oh.. my.. god, why didn't I think to try that, sorry about that! The swipe gesture was just a thought (similar to how Edge on Android switches tabs), but it's good to know systray is planned! As I said, it's pretty impressive already, I'm using it on my TMAX TM101W638L and I really didn't think tablet mode could get much better, but you managed it!

**Comment by ashelleyPurdue** [comment]
It'd be nice if a long press weren't required.

**Comment by Iamtheboneofmysword** [comment]
I think it would be better to show all the notification icons of the task bar.

**Comment by mdtauk** [comment]
> I think it would be better to show all the notification icons of the task bar.

It wouldn't be a Mobile Shell if you included the taskbar.  This is the equivalent of the NavigationBar Windows Phone had

**Comment by Iamtheboneofmysword** [comment]
> > I think it would be better to show all the notification icons of the task bar.
> 
> It wouldn't be a Mobile Shell if you included the taskbar. This is the equivalent of the NavigationBar Windows Phone had

You are right. I use the Shell on my surface to replace the Tablet Mode of Win10. It looks good and I hope it can make Windows touch screen devices to be real tablets. Notification icons may be placed at top left corner if possible.

**Comment by TyJOrtiz** [comment]
@ADeltaX it's not that hard to do, right? I just did it with your code

**Comment by ADeltaX** [comment]
"it's not that hard to do, right?"

lol

*no comment*
